cmd/cmd.go

```diff
@@ -879,6 +879,24 @@ func NewCLI() *cobra.Command {
 		Args:    cobra.ExactArgs(0),
 		RunE:    RunServer,
 	}
+	serveCmd.SetUsageTemplate(serveCmd.UsageTemplate() + `
+Environment Variables:
+
+	OLLAMA_HOST
+		The host:port to bind to (default "127.0.0.1:11434")
+
+		Examples:
+			"127.0.0.1:11434"
```

Comment:
not sure if we need more examples but other possible values are just the host `127.0.0.1` (default to port 11434), just the port `:11434` (default to host 127.0.0.1)